data/tlg5034/tlg001a/tlg5034.tlg001a.perseus-grc1.xml

```diff
@@ -8,7 +8,7 @@
                 <title>Scholia in Pindarum Olympian Odes</title>
```

Comment:
Add an `xml:lang="eng"` attribute to the `<teiHeader>` element. And then add `xml:lang="mul"` to this `<title>` element.